### ISSUE TITLE
Pipe mkv

### ISSUE THREAD

**Issue by FallingSnow** [fluent-ffmpeg/node-fluent-ffmpeg#611]
I am trying to pipe the output to a `fs.createWriteStream` but I keep getting `Output format mkv is not available`. My code:

```javascript
_self.outputStream = fs.createWriteStream(_self.output.path);
_self.ffmpegCommand.format('mkv').pipe(_self.outputStream);
```
The rest of the command has been set up properly. If I change the above lines to
```javascript
_self.ffmpegCommand.save(_self.output.path);
```
Then everything works. I'm on Windows x64 so idk if that is the problem. Perhaps you can only use pipes on Unix?

**Comment by njoyard** [comment]
Can you post ffmpeg stderr please (in both cases) ?

**Comment by FallingSnow** [comment]
I believe I found the problem. Format should be `matroska` not `mkv`. Sorry to have wasted your time.

Also I switched over to encoding into `bmp`. I'm using the following setup:
```javascript
_self.ffmpegCommand
.videoCodec('bmp')
.format('image2')
.outputOptions('-vf', 'fps=fps=1')
.pipe(_self.outputStream);
```
I am then using [bmp-js](https://github.com/shaozilee/bmp-js) to try to decode the bmp on the other side of the pipe but I keep getting index out of range errors. Perhaps this is something I should ask the author though.

Thank you for taking the time to respond.
